src/lib/core/time_utils.py

```diff
@@ -1,5 +1,6 @@
 from datetime import datetime
 import time
+import dateutil
 
```

Comment:
Wouldn't it be better to name this module datetime_utils? Seems that it works with datetime, not just time :)

Reply:
It's a python built-in module :) Can't rename :)

Reply:
Nastassia's probably meaning this very file she's reviewing

Reply:
Oh, sorry. Silly me :)
Fixed. Renamed to datetime_utils.py

Reply:
yes, meant exactly this, thanks! unfortunatelly cannot put comment on the file name :)